runtime/laos/Cargo.toml

```diff
@@ -149,6 +150,7 @@ std = [
 	"pallet-balances/std",
 	"pallet-elections-phragmen/std",
 	"pallet-session/std",
+	"pallet-democracy/std",
```

Comment:
Not related to this line but the requested modification affects this file. We should replace
```toml
fast-mode=[]
```
for:
```toml
fast-runtime=[]
```
This is what `prod_or_fast` does: https://github.com/paritytech/polkadot-sdk/blob/a947cb831d8d56228e7aa84355410c50864a816e/polkadot/runtime/common/src/lib.rs#L272